api/ca.proto

```diff
@@ -34,11 +34,15 @@ message NodeCertificateStatusResponse {
 }
 
 message IssueNodeCertificateRequest {
-	NodeRole role = 1;
+	// DEPRECATED: Role is now selected based on which secret is matched.
+	NodeRole role = 1 [deprecated=true];
```

Comment:
That's correct. The fields are left in as deprecated to avoid accidentally reusing the field numbers.